rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -1057,7 +1153,7 @@ public void setUserSettings(String refreshTime) {
     /**
      * Change the currently selected node
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "cause" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)